src/composables/use-media-query.ts

```diff
@@ -66,21 +70,28 @@ export function useMediaQuery(
 
   return matches
 }
-
+type BreakpointWithoutXs = Exclude<Breakpoint, 'xs'>
 /**
  * Check whether the current screen meets
  * or exceeds the provided breakpoint size.
  */
-export const isMinScreen = (breakpointName: Breakpoint, options?: Options) => {
+export const isMinScreen = (
+  breakpointName: MaybeComputedRef<Breakpoint>,
+  options?: Options
+) => {
   if (breakpointName === 'xs') {
```

Comment:
Does this also need `unref`?

Reply:
Thank you for catching this! I don't know why there was no warning.